src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+function createNotification(messange, type) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = type; // // success або error
+  notification.textContent = messange;
```

Comment:
The variable `messange` is used here, which contains a typo. It should be `message` to match the intended parameter name.